content/release.rst

```diff
@@ -37,6 +37,21 @@ Such a release can have a normal version... and if changes are made
 during the Q&A review a new patch version is released.
```

Comment:
Q&A means Questions And Answers.... QA means Quality Assurance. I think here we are referring to a QA review